amp-story/templates/ampmars/index.html

```diff
@@ -0,0 +1,967 @@
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>The Mars Cruise</title>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+    <link rel="canonical" href="index.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Tienne:700" rel="stylesheet">
+    <script type="application/ld+json">
+      {
+        "bookendVersion": "v1.0",
+        "@context": "http://schema.org",
+        "@type": "CreativeWork",
+        "headline": "The Mars Cruise",
+        "url": "https://www.example.com"
+      }
+    </script>
+    <style amp-custom>
+      :root {
+        --yellow: #ffd200;
+        --bl-0: rgba(0, 0, 0, 0);
```

Comment:
Can we use more human-readable variable names for these two variables?

Reply:
Haha, yes, I would love to.  The linter was complaining about the line being over 100 characters ([L:96](https://github.com/hookerz/samples/blob/amp-story-mars/amp-story/templates/ampmars/index.html#L96)).  I was trying to get the linter to pass and was having a mess of a time with it trying multiline.

Reply:
Looks like it's pretty far from 100 characters?  I can try to fix the multiline issues, but in the meantime can we pick a name that fits?  Shouldn't there be 70-something characters left on that line?

Reply:
[L:96](https://github.com/hookerz/samples/blob/amp-story-mars/amp-story/templates/ampmars/index.html#L96) is currently 99 characters.

Reply:
@normmcgarry can you change these names to be meaningful and simply ignore that specific lint check for right now?